Makefile

```diff
@@ -134,7 +135,7 @@ container-release: container
 	docker push $(DOCKER_REPO):latest
 
 .PHONY: integration-test-dependencies
-integration-test-dependencies: $(THANOS) $(UP)
+integration-test-dependencies: $(THANOS) $(UP) $(DEX)
```

Comment:
We might actually want to rename this to 
```suggestion
test-integration-dependencies: $(THANOS) $(UP) $(DEX)
```
as the actual test is run with `test-integration`. Just an inconsistency I noticed.

Reply:
Let's do that in a follow up since it's a pre-existing thing to clean up